lib/src/runner/configuration/load.dart

```diff
@@ -49,13 +49,25 @@ Configuration load(String path, {bool global: false}) {
 
   if (document.value == null) return Configuration.empty;
 
-  if (document is! Map) {
-    throw new SourceSpanFormatException(
-        "The configuration must be a YAML map.", document.span, source);
+  if (document is YamlMap) {
+    var loader = new _ConfigurationLoader(document, source, global: global);
+    var config = loader.load();
+    var includePathNode = document.nodes["include"];
+    if (includePathNode != null && includePathNode.value is! String) {
+      throw new SourceSpanFormatException(
+          "The 'include' field must be omitted or a String",
+          includePathNode.span,
+          source);
+    }
+    if (includePathNode != null) {
+      var basePath = p.join(p.dirname(path), includePathNode.value);
```

Comment:
I just realized: this should parse `path` as a URL, to ensure that it's cross-platform. We should also document that it's a relative `file:` URL.

Reply:
Done.